cypress/integration/channel/edit_message_spec.js

```diff
@@ -61,4 +61,39 @@ describe('Edit Message', () => {
         // * Assert emoji picker is not visible
         cy.get('#emojiPicker').should('not.exist');
     });
+
+    it('Edit modal should open for the first post when the second post is still pending', () => {
```

Comment:
```suggestion
    it('M13542 Open edit modal immediately after making a post', () => {
```

Please update the description to include the test key M13542.  We recently used that to track between codebase and list of test cases.